### ISSUE TITLE
Introduce `target_list=` parameter to Qubes RPC Qrexec Policy

### ISSUE THREAD

**Issue by ben-grande** [QubesOS/qubes-issues#8731]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### The problem you're addressing (if any)

Currently, when using the `ask` resolution, when specifying the `default_target=` parameter, it requires the `target=` parameter, thus restricting to only one target instead of a list:
```
service * @anyvm @anyvm defaul_target=qube1 target=qube1
```

But what if I want a target pre-select but not limit the target list to it? Currently not possible.

### The solution you'd like

As `target=` can't have multiple values especially because of `allow` where only one target needs to match, I would like to propose for the implementation of a new parameter. The `target_list=`.

This parameter should accept domains and keywords that are accepted in the destination field, such as `@tag`.

### The value to a user, and who that user might be

If the following is implemented:
- https://github.com/QubesOS/qubes-issues/issues/5397

In combination with this issue, it would help users target policies with ask to only vms that support a certain service:
```
qubes.GitFetch +dir @anyvm @default default_target=sys-git target_list=@features:supported-rpc:qubes.GitFetch
```

The dialog would pre-select `sys-git` and from the available list will be shown all qubes that report they have the service `qubes.GitFetch` available.

**Comment by marmarek** [comment]
> when specifying the `default_target=` parameter, it requires the `target=` parameter,

No, it doesn't require it. The list of allowed targets is collected from other rules. You can use it this way:
```
service * @anyvm qube1 ask
service * @anyvm qube2 ask
service * @anyvm qube3 ask
service * @anyvm @default ask default_target=qube1
```
Then, when making the qrexec call, do not specify target in the source qube at all (use `@default` keyword, or empty string).
You can also add `default_target=qube1` to all the ask actions, to have uniform handling, but in practice that's rarely useful IMO - it's easier to skip actual target when making the call.

**Comment by ben-grande** [comment]
To be more specific, it is only useful when the destination is `@default`.

Currently there is not way to get a list of target qubes when destination is `@default`.

If doing this:
```
service * @anyvm @default ask default_target=qube1
```

The `qube1` will be the only qube available to be chosen.

I always use `@default` as destination.

**Comment by ben-grande** [comment]
> See my example above

I saw it and read the documentation many times, but this moment the lamp behind my head turned on.

Thanks for your time, I understand know.
